### ISSUE TITLE
effect sounds are too loud

### ISSUE THREAD

**Issue by 0rangeGhost** [berkayylmao/NFS-Chat-Chaos-Mod#14]
in compare to the volume level of the game most effect's sounds are much louder, so i have to turn down volume (don't count earrape =) )
i would like to have them 1.5-2x quiter

**Comment by berkayylmao** [comment]
I agree, never bothered leveling the audio on the sound effects. Will push the fix on the next version and tag you here.

**Comment by berkayylmao** [comment]
All sound are normalized in commit 1488e8f6200d43856e10fa0cab0b4d3b113964ab. Although the sound effects themselves are not pushed into the repo, new audio files will be present in the next version.
